exercises/acronym/src/example.c

```diff
@@ -0,0 +1,50 @@
+#include <ctype.h>
+#include <stdlib.h>
+
+static int count_words(const char phrase[]);
+static int is_first_letter_of_word(int c, int b);
```

Comment:
Yeah, I used to work like that for the same reason, but grew really tired of updating static prototypes. Additionally, I realized that a decent IDE will give you an index of all of your static functions without any tedious bookkeeping by the developer.